packages/wonder-blocks-button/src/components/__docs__/navigation-callbacks.stories.mdx

```diff
@@ -44,16 +45,24 @@ Which prop you choose depends on your use case.
   callback function passed in to this prop, and Wonder Blocks will run
   the async code in parallel with client-side navigation or while opening
   a new tab, but will wait until the async code finishes to start a
-  server-side navigation. This prop should be used when it's okay to load
+  server-side navigation. If the promise rejects the navigation will
+  happen anyway. This prop should be used when it's okay to load
   the next URL while the async callback code is running.
 
 This table gives an overview of the options:
 
-| Prop         | Async safe? | Completes before navigation? |
-|--------------|-------------|------------------------------|
-| onClick      | no          | yes                          |
-| beforeNav    | yes         | yes                          |
-| safeWithNav  | yes         | no                           |
-
+| Prop        | Async safe? | Completes before navigation? |
+|-------------|-------------|------------------------------|
+| onClick     | no          | yes                          |
+| beforeNav   | yes         | yes                          |
+| safeWithNav | yes         | no                           |
 
+It is possible to use more than one of these props on the same element.
+If multiple props are used, they will run in this order: first `onClick`,
+then `beforeNav`, then `safeWithNav`. If both `beforeNav` and `safeWithNav`
+are used, the `safeWithNav` callback will not be called until the
+`beforeNav` promise resolves successfully. If the `beforeNav` promise
+rejects, `safeWithNav` will not be run.
```

Comment:
My vague recollection of the props was incorrect.  While we are using disjoint props, it's `_target` and `beforeNav` that can't be used together.

Thanks for making it explicit that these props can be used together and the ordering in which they're run.